exemplar/src/main/kotlin/com/squareup/exemplar/HelloWebAction.kt

```diff
@@ -13,8 +10,11 @@ import javax.inject.Singleton
 class HelloWebAction : WebAction {
     @Get("/hello/{name}")
     @ResponseContentType(MediaTypes.APPLICATION_JSON)
-    fun hello(@PathParam name: String, @RequestHeaders headers: Headers): HelloResponse {
-        return HelloResponse("YO", name.toUpperCase())
+    fun hello(@PathParam name: String, @RequestHeaders headers: Headers,
```

Comment:
Format when wrapping is to put each parameter on its own line, with the return type on a separate line. e.g.

````kotlin
fun hello(
   @PathParam name: String,
   @RequestHeaders headers: Headers,
   @QueryParam("nickname") nickname: String?
) : HelloResponse {
}


````

Reply:
Gotcha, will do